.code-samples.meilisearch.yaml

```diff
@@ -423,25 +423,27 @@ documents_guide_add_movie_1: |-
       title: "Amélie Poulain".to_string(),
     }
   ], None).await.unwrap();
-document_guide_create_index_primary_key: |-
-  client.create_index("movies", Some("reference_number")).await.unwrap();
-document_guide_add_document_primary_key: |-
+primary_field_guide_update_document_primary_key: |-
+  client.index("books").update(Some("title")).await.unwrap();
+primary_field_guide_create_index_primary_key: |-
+  client.create_index("books", Some("reference_number")).await.unwrap();
+primary_field_guide_add_document_primary_key: |-
   #[derive(Serialize, Deserialize)]
-  struct Movie {
-    id: String,
+  struct Book {
+    reference_number: String,
     title: String,
-    poster: String,
-    overview: String,
-    release_date: String
+    author: String,
+    genres: Vec<String>,
+    price: i64
   }
 
-  let task: Task = client.index("movies").add_documents(&[
-    Movie {
-      "reference_number": "287947".to_string(),
-      "title": "Shazam".to_string(),
-      "poster": "https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg".to_string(),
-      "overview": "A boy is given the ability to become an adult superhero in times of need with a single magic word.".to_string(),
-      "release_date": "2019-03-23".to_string()
+  let task: Task = client.index("books").add_documents(&[
+    Book {
+      reference_number: "287947".to_string(),
+      title: "Diary of a Wimpy Kid".to_string(),
+      author: "Jeff Kinney".to_string(),
+      genres: vec!["comedy".to_string(),"humor".to_string()],
+      price: 5.00
```

Comment:
I'm pretty sure that this value will be considered as a float and an `i64` is declared will it still work.

Reply:
I updated it to f64